integration_tests/tests/temporary-accommodation/manage/bedspaceSearch.cy.ts

```diff
@@ -148,7 +148,7 @@ context('Bedspace Search', () => {
     Page.verifyOnPage(BedspaceShowPage, premises, room)
   })
 
-  it('allows me to view an overlapping booking', () => {
+  it('allows me to view an overlapping offenders referral', () => {
```

Comment:
```suggestion
  it('allows me to view an overlapping offender's referral', () => {
```
```suggestion
  it('allows me to view overlapping offender referrals', () => {
```

or something? or netiher?